sections/futures/LeverageSlider/LeverageSlider.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import Slider from '@material-ui/core/Slider';
+import imageBackground from 'assets/svg/futures/sliderbackground.svg';
+
+type LeverageSliderProps = {
+	min?: number;
+	max?: number;
+	value?: number;
+	defaultValue?: number;
+	disabled?: boolean;
+	onChange: (event: React.ChangeEvent<{}>, value: number | number[]) => void;
+	onChangeCommitted: (event: React.ChangeEvent<{}>, value: number | number[]) => void;
+};
+
+const LeverageSlider: React.FC<LeverageSliderProps> = ({
+	min,
+	max,
+	value,
+	defaultValue,
+	disabled,
+	onChange,
+	onChangeCommitted,
+}) => {
+	return (
+		<LeverageSliderContainer>
+			<div>
+				<StyledSlider
+					min={min || 0}
+					max={max || 10}
+					step={0.1}
+					defaultValue={defaultValue || 1}
+					value={value}
+					onChange={onChange}
+					onChangeCommitted={onChangeCommitted}
+					disabled={disabled}
+					marks={[
+						{ value: 1, label: '1x' },
+						{ value: 10, label: '10x' },
+					]}
+					valueLabelDisplay="on"
+					valueLabelFormat={(v) => `${v}x`}
+					$currentMark={value || 1}
+				/>
+			</div>
+		</LeverageSliderContainer>
+	);
+};
+
+const LeverageSliderContainer = styled.div`
+	width: 334px;
+	height: 24px;
+	background: url(${imageBackground as any}) no-repeat;
+	padding: 0 19px 0 4px;
+	box-sizing: border-box;
+`;
+
+const StyledSlider = styled(Slider)<{ $currentMark?: number }>`
+	color: transparent !important;
+
+	.MuiSlider-rail {
+		margin-top: -11px;
+		background-color: transparent;
+		height: 24px;
+		left: 0;
+		right: 0;
+	}
+
+	.MuiSlider-track {
+		height: 4px;
+		background-color: #7d6b54;
```

Comment:
Not sure if we wanted to make these colors reuse-able in the theme file?

Reply:
Yeah. The plan is to move all the hardcoded colors out in a separate PR. I didn't want to affect other components dependent on some of the colors.